simtools/corsika/corsika_output.py

```diff
@@ -976,44 +1036,416 @@ def num_photons_per_event(self):
         self._num_photons_per_event = np.sum(self.num_photons_per_event_per_telescope, axis=0)
         return self._num_photons_per_event
 
-    def get_num_photons_distr(self, bins=50, hist_range=None, event_or_telescope="event"):
+    def get_num_photons_per_event_distr(self, bins=50, hist_range=None):
         """
-        Get the distribution of photons per event or per telescope, depending on the string
-        passed through `event_or_telescope`.
+        Get the distribution of photons per event.
 
         Parameters
         ----------
         bins: float
             Number of bins for the histogram.
         hist_range: 2-tuple
             Tuple to define the range of the histogram.
-        event_or_telescope: str
-            Indicates if the distribution of photons is given for the events, or for the telescopes.
-            Allowed values are: "event" or "telescope".
-
 
         Returns
         -------
         numpy.ndarray
             The counts of the histogram.
         numpy.array
             Number of photons per event.
+        """
+        hist, edges = np.histogram(self.num_photons_per_event, bins=bins, range=hist_range)
+        return hist.reshape(1, bins), edges.reshape(1, bins + 1)
 
+    def get_num_photons_per_telescope_distr(self, bins=50, hist_range=None):
+        """
+        Get the distribution of photons per telescope.
 
-        Raises
-        ------
-        ValueError:
-            if event_or_telescope not valid.
+        Parameters
+        ----------
+        bins: float
+            Number of bins for the histogram.
+        hist_range: 2-tuple
+            Tuple to define the range of the histogram.
+
+        Returns
+        -------
+        numpy.ndarray
+            The counts of the histogram.
+        numpy.array
+            Number of photons per telescope.
         """
-        if event_or_telescope == "event":
-            hist, edges = np.histogram(self.num_photons_per_event, bins=bins, range=hist_range)
-        elif event_or_telescope == "telescope":
-            hist, edges = np.histogram(self.num_photons_per_telescope, bins=bins, range=hist_range)
-        else:
-            msg = "`event_or_telescope` has to be either 'event' or 'telescope'."
-            self._logger.error(msg)
-            raise ValueError
-        return hist, edges
+
+        hist, edges = np.histogram(self.num_photons_per_telescope, bins=bins, range=hist_range)
+        return hist.reshape(1, bins), edges.reshape(1, bins + 1)
+
+    def export_histograms(self, output_dir="./"):
+        """
+        Export the histograms to ecsv files.
+
+        Parameters
+        ----------
+        output_dir: str
+            Output directory, where to save the histograms.
+        """
+
+        self._export_1D_histograms(output_dir=output_dir)
+        self._export_2D_histograms(output_dir=output_dir)
```

Comment:
I added the test, which is simply a repetition of the previous two. Ideally we mock the test, but I would like to avoid spending time with that at this point